strr-host-pm-web/app/stores/document.ts

```diff
@@ -228,7 +232,8 @@ export const useDocumentStore = defineStore('host/document', () => {
         method: 'POST',
         body: formData
       })
-
+      // api doesn't give documentType back
+      res.documentType = uiDoc.type
```

Comment:
Is this necessary if the apiDocuments computed above adds the documentType already? or maybe the computed could be simplified to just 

```
const apiDocuments = computed(() => storedDocuments.value.map(item => item.apiDoc))
```

Also my bad with the documentType property name

Reply:
Nice catch, I will update the computed since the api validation requires 'documentType' so it should be with the 'apiDoc' always (the api response when we get the application will include 'documentType' in the apiDoc so this will keep it the same)